src/run.rs

```diff
@@ -100,15 +101,16 @@ pub async fn run(
     match selected_scenario {
         None => {
             println!("Non-existing scenario!");
-            return Err(())
-        },
+            return Err(());
+        }
         Some(scenario) => {
-            let fixtures_in_directories: Vec<String> = scenario
+            let mut fixtures_in_directories: Vec<String> = scenario
                 .directories
                 .iter()
                 .filter_map(extract_fixtures_from_directory)
                 .flatten()
                 .collect();
+            fixtures_in_directories.sort();
```

Comment:
dump question: Cannot you pipe the `sort` after the collect?  Or vectors don't allow this?
Maybe using [sorted](https://docs.rs/itertools/0.8.0/itertools/trait.Itertools.html#method.sorted)

Reply:
Cool, I'm not much into rust so I missed that. I'll try it